combineDistanceThreshold: 0.5
pdf_std: 400
confidenceThreshold: 0.1
NumOfParticles: 3500
NumberOfParticlesToReset: 20
windDir: 0

mapPercent: 0.0025
percentTheta: 0.0055
percentQ: 0.015
percentV: 0.01
percentDy: 0.01
percentDz: 0.01

X_u: 4.2
X_l: 0

Y_u: 2.67
Y_l: 0

Z_u: 0.2
Z_l: 0.2

Theta_range: 0.523599
Theta_u: 0.523599
Theta_l: -0.523599

Q_u: 1.391346e-04
Q_l: 1.391346e-04

V_u : 3
V_l : 2

Dy_u : 5.020525e-02
Dy_l : 1.020525e-02

Dz_u : 5.222726e-09
Dz_l : 3.222726e-09
